book/introduction/tooling.md

```diff
@@ -1,23 +1,23 @@
 ## Chapter iii - Tooling
 
-In order to make our roguelike we'll be using a few different tools. First off, we'll be using Ruby. I've chosen Ruby for this initial edition for a couple of reasons. First and foremost, I find Ruby to be fairly easy to understand and when I want to build something quickly it's my goto. Second, I feel like object-oriented programming lends itself fairly well for programming games. Lastly, it has Ncurses bindings available through the gem `curses`.
+In order to make our roguelike we'll be using a few different tools. First off, we'll be using Ruby. I've chosen Ruby for this initial edition for a couple of reasons. First and foremost, I find Ruby to be fairly easy to understand and when I want to build something quickly it's my goto. Second, I feel like object-oriented programming lends itself fairly well for programming games. Lastly, it has ncurses bindings available through the gem `curses`.
 
-What is Ncurses you ask? Ncurses stands for New Curses, it's a freeware reimplementation of the original curses library. Its purpose is to make managing screen state easier and more portable. If you've ever installed some flavor of linux on a computer you might have seen something like this:
+What is ncurses you ask? ncurses stands for *new curses*, it's a freeware reimplementation of the original curses library. Its purpose is to make managing screen state easier and more portable. If you've ever installed some flavor of Linux on a computer you might have seen something like this:
 
 \includegraphics[width=\linewidth]{images/ncurses-example.png}
 
-This screen was created using Ncurses. So why do **we** need Ncurses? After all, we're just writing a simple game. Using Ruby let's try to do something simple like clearing the screen. If you're on OSX you might have written something like:
+This screen was created using Ncurses. So why do *we* need Ncurses? After all, we're just writing a simple game. Using Ruby let's try to do something simple like clearing the screen. If you're on macOS you might have written something like:
```

Comment:
The references to "Ncurses" want to be changed to "ncurses" for consistency, no?

Reply:
Good catch. Fixed these and one more occurrence in 80e682f7729c9e7b25a0d44d1594d244c7d9042b.